server/src/main/java/org/opensearch/action/search/SearchRequestOperationsListener.java

```diff
@@ -45,6 +62,7 @@ static final class CompositeListener extends SearchRequestOperationsListener {
         CompositeListener(List<SearchRequestOperationsListener> listeners, Logger logger) {
             this.listeners = listeners;
             this.logger = logger;
+            this.setEnabled(true);
```

Comment:
This doesn't seem right. Why are setting this here? Considering CompositeListener is a list of listeners, and this property(ie enabled) is present on individual listeners. So doesn't make sense to add it here.

Reply:
yeah, we probably won't use this flag. Let me remove it.